src/components/lobby/FilterBox.js

```diff
@@ -0,0 +1,150 @@
+import React, { useRef, useState } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { Popup, Input, Icon } from "semantic-ui-react";
+
+import styles from "./FilterBox.module.scss";
+import { GameType } from "config/propTypes";
+
+const sameWidthModifier = {
+  name: "sameWidth",
+  enabled: true,
+  fn({ state }) {
+    state.styles.popper.width = `${state.rects.reference.width}px`;
+  },
+  phase: "beforeWrite",
+  requires: ["computeStyles"],
+};
+
+const propTypes = {
+  games: PropTypes.arrayOf(GameType).isRequired,
+  filters: PropTypes.shape({
+    query: PropTypes.string,
+    games: PropTypes.arrayOf(GameType),
+  }).isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+const FilterBox = ({ games, filters, onChange }) => {
+  const [open, setOpen] = useState(false);
+  const searchRef = useRef();
+
+  const selectedGames = filters.games;
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setOpen(false);
+  };
+
+  const handleClose = (e) => {
+    const searchInput = searchRef?.current?.inputRef?.current;
+    if (!searchInput || e.target !== searchInput) {
+      setOpen(false);
+    }
+  };
+
+  const handleToggleGame = (game) => {
+    let { games } = filters;
+    if (selectedGames.includes(game)) {
+      if (selectedGames.length === 1) return;
```

Comment:
Alternatively, deselecting all games could disable this filter = automatically select all games

Reply:
done 👍